queue_services/entity-emailer/src/entity_emailer/worker.py

```diff
@@ -103,15 +103,18 @@ def send_email(email: dict, token: str):
         logger.debug('Send email: email object(s) is missing')
         raise QueueException('Unsuccessful sending email - required email object(s) is missing. ')
 
-    resp = requests.post(
-        f'{APP_CONFIG.NOTIFY_API_URL}',
-        json=email,
-        headers={
-            'Content-Type': 'application/json',
-            'Authorization': f'Bearer {token}'
-        }
-    )
-    if resp.status_code != HTTPStatus.OK:
+    try:
+        resp = requests.post(
+            f'{APP_CONFIG.NOTIFY_API_URL}',
+            json=email,
+            headers={
+                'Content-Type': 'application/json',
+                'Authorization': f'Bearer {token}'
+            }
+        )
+        if resp.status_code != HTTPStatus.OK:
+            raise EmailException
+    except Exception:  # noqa B902; pylint: disable=W0703; we don't want to fail out the email, so ignore all.
```

Comment:
Any idea what exception occurs in the ticket scenario? Maybe we should check for that specifically?

Or, maybe, continue only if the response status code is "OK"?

Reply:
Its max retry error https://app.zenhub.com/workspaces/entities---olga-65af15f59e89f5043c2911f7/issues/gh/bcgov/entity/23413#issuecomment-2362276393
But we should any unexpected error

Reply:
I agree that we should handle any error. I guess this is already the cleanest code to do that.